mopidy/internal/validation.py

```diff
@@ -17,9 +17,14 @@
     "genre",
     "date",
     "comment",
-    "any",
+    "disc_no",
+    "musicbrainz_albumid",
+    "musicbrainz_artistid",
+    "musicbrainz_trackid",
```

Comment:
Need to update the types for these following https://github.com/mopidy/mopidy/pull/1842/files